python/paddle/fluid/dygraph/dygraph_to_static/program_translator.py

```diff
@@ -300,16 +300,17 @@ def __call__(self, *args, **kwargs):
         """
 
         # 1. call dygraph function directly if not enable `declarative`
-        if not self._program_trans.enable_declarative:
+        if not self._program_trans.enable_static:
```

Comment:
maybe `enable_to_static` is better? keep consistent with `paddle.jit.to_static`

because we have `paddle.disable_static`, users may be confused whether it is related to `enable_static` here

Reply:
Done